assets/js/scripts.js

```diff
@@ -1,8 +1,12 @@
 /*
 
-SitePanel Scripts
+Design System Scripts
 
-    This document contains all of the SitePanel-specific front-
+    This document contains all of the design system-specific front-
     end scripts.
 
 */
+
+$( ".menutoggle" ).click(function() {
```

Comment:
Another way to write this that will limit event handlers, possibly improving performance:
```
$("body").on("click", ".menutoggle", function () {
    $(this).toggleClass("show");
});
```